### PR TITLE
Refactor software lab

### PR DESCRIPTION
**Pull request by phmkopp**

### CODE REVIEW

**Comment by phmkopp** [comment]
Main changes:
* Apply vtu11 code style
* Refactor pwrite_pyramids3D_test
* Create helper function for reading testfiles in vtu11_testing.hpp / .cpp
* Revert changes that do nothing else than adding/removing whitespaces

File: test/vtu11_testing.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    std::string contents, str;
+
+    while( std::getline( file, str ) )
+    {
+        contents += str + "\n";
+    }
+
+    file.close( );
```
Comment (philbucher) [inline test/vtu11_testing.cpp:34]: I think this is implicitly closed at the end of the scope but I would leave it for readability

**Review by philbucher — COMMENTED** [review]
nice job looks clean & consistent to me!

**Review by philbucher — APPROVED** [review]

File: test/vtu11_testing.cpp (hunk 1 above)
Comment (phmkopp) [reply test/vtu11_testing.cpp:34]: Okay, interesting, I didn't know. But I never got into trouble when I forgot closing it, so that would explain why :)

**Pull request merged**